monitoringcontroller/monitoringcontroller.go

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2022 Renesas Electronics Corporation.
+// Copyright (C) 2022 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitoringcontroller
+
+import (
+	"context"
+
+	"github.com/aoscloud/aos_common/api/cloudprotocol"
+	"github.com/aoscloud/aos_communicationmanager/config"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// MonitoringSender sends monitoring data.
+type MonitoringSender interface {
+	SendMonitoringData(monitoringData cloudprotocol.MonitoringData) (err error)
+}
+
+// MonitorController instance.
+type MonitorController struct {
```

Comment:
`monitoringcontroller.MonitorController` looks not so good. Consider renaming to just `Controller` or `Instance`.